composer/devices/device.py

```diff
@@ -31,6 +31,7 @@ class Device(Serializable, ABC):
 
     dist_backend: str = ''
     name: str = ''
+    _device = None
```

Comment:
why do we need this?

Reply:
Otherwise we get a pylint issue:

```  /workdisk/brandon/composer_git/composer/core/state.py:952:50 - error: Cannot access member "_device" for type "Device"
    Member "_device" is unknown (reportGeneralTypeIssues)
```